src/date-time/date-time.tsx

```diff
@@ -25,59 +25,60 @@ const StyledSuffix = StyledDateText;
 
 const defaultDateFormat = "MMMM d yyyy, h:mmaaaaa'm'";
 
-const ThemelessDateTime: React.FC<DateTimeProps> = ({
-  date,
-  dateFormat = defaultDateFormat,
-  prefix,
-  suffix,
-  overrides = {},
-}) => {
-  const theme = useTheme();
+const ThemelessDateTime = React.forwardRef<HTMLParagraphElement, DateTimeProps>(
```

Comment:
I think it shoud be a DIV element
```suggestion
const ThemelessDateTime = React.forwardRef<HTMLDivElement, DateTimeProps>(
```